java/vector/src/main/java/org/apache/arrow/vector/NullableBigIntVector.java

```diff
@@ -0,0 +1,299 @@
+/*******************************************************************************
+
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
maybe minor but I usually sometimes see a different format for the license:

```
/**
 * Licensed to the Apache Software Foundation (ASF) under one
 * ...
 *\
```
It would be good to be consistent, but maybe that would be better to address separately after all this

Reply:
Okay will look into it at the end.